rules/proxy/proxy_exchange_owassrf_exploitation.yml

```
@@ -0,0 +1,25 @@
+title: Potential OWASSRF Exploitation Attempt - Proxy
+id: 1ddf4596-1908-43c9-add2-1d2c2fcc4797
+status: experimental
+description: Detects exploitation attempt of the OWASSRF variant targeting exchange servers It uses the OWA endpoint to access the powershell backend endpoint
+references:
+    - https://www.crowdstrike.com/blog/owassrf-exploit-analysis-and-recommendations/
+    - https://www.rapid7.com/blog/post/2022/12/21/cve-2022-41080-cve-2022-41082-rapid7-observed-exploitation-of-owassrf-in-exchange-for-rce/
+author: Nasreddine Bencherchali
+date: 2022/12/22
+tags:
+    - attack.initial_access
+    - attack.t1190
+logsource:
+    category: proxy
+detection:
+    selection:
+        # Look for the header: X-OWA-ExplicitLogonUser: owa/mastermailbox@outlook.com
+        cs-method: 'POST'
+        c-uri|contains|all:
+            - '/owa/mastermailbox'
```

```suggestion
            - '/owa/'
```
the generic payload : `https://{exchange_host}/owa/{email_address]/powershell`

---

can add a `@` for the email